### ISSUE TITLE
Some kind of new abilities

### ISSUE THREAD

**Issue by F1reStar** [AlexModGuy/Ice-and-Fire-Suggestions#907]
Hi, I have been playing a lot of I&F lately and thought that each kind of dragon-related category (Ice, Fire, and Lightning), are kind of similar. They mostly have just different textures and slightly different abilitie, I though a way to make each of the three types of dragon items more unique and different to each other is to have a new set of items that give a unique advantage and favour a specific playstyle. These could be a new armour slot or something, like maybe a ring or bracelet or belt or necklace or something else.
-The ice item could favour a defensive playstile, slowing and freezing enemies, and creative defences such as shields, walls, or armour to protect the wearer.
-The fire item could favour an offensive playstile, making anything you hit get set on fire, and making you run faster by creating a trail of fire behind you that lasts for a limited time before disappearing, and can burn any mob or player that walks into it.
-The lightning item could favour an elusive or mixed playstyle, allowing some kind of phase shift or teleportation, but not like normal teleportation, instead it could only teleport you ten blocks ahead but with almost no cooldown, it could also have a small chance to electricute (stun) any mob or player you hit, making them unable to move or attack or do anything for a very short period of time.

TLDR: I think each kind of dragon-magic related category (Ice, Fire, and Lightning), could be more unique and different, so I propose items that favour specific playstyles: ice=defensive, fire=offensive, lightning=elusive.

Thanks!
